erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md

```diff
@@ -0,0 +1,64 @@
+# Promotional Scheme
+
+Promotional schemes are the key to successful retail stores. Promotional schemes can be easily configure in the erpnext. Promotional scheme is linked to the pricing rule, against each slab system will generate the pricing rule.
```

Comment:
Promotional schemes help retail stores become successful. They can be easily configure in ERPNnext. A Promotional scheme is linked to a pricing rule, against each slab system will generate the pricing rule.